src/window-rpc/setup-window-rpc.ts

```
@@ -178,9 +178,14 @@ export function setupWindowRpc() {
 						})
 					},
 				})
+				const claimResponses: CreateClaimResponse[] = []
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)